src/stories/containers/Finances/components/HeaderTable/HeaderQuarterly/CellQuarterly.tsx

```diff
@@ -0,0 +1,152 @@
+import styled from '@emotion/styled';
+import { returnShortNameForMetric } from '@ses/containers/Finances/utils/utils';
+import { useThemeContext } from '@ses/core/context/ThemeContext';
+import lightTheme from '@ses/styles/theme/light';
+import React from 'react';
+import type { MetricsWithAmount } from '@ses/containers/Finances/utils/types';
+import type { WithIsLight } from '@ses/core/utils/typesHelpers';
+
+interface Props {
+  metrics: MetricsWithAmount[];
+  quarterly: string;
+  isTotal?: boolean;
+  className?: string;
+}
+
+export const CellQuarterly: React.FC<Props> = ({ metrics, quarterly, isTotal = false, className }) => {
+  const { isLight } = useThemeContext();
+
+  return (
+    <MainContainer isLight={isLight} isTotal={isTotal} className={className}>
+      <ContainerCell>
+        <Quarterly isLight={isLight}>{quarterly}</Quarterly>
+        <ContainerMetricsData>
+          {metrics?.map((metric, index) => (
+            <Metrics key={index}>
+              <Name isLight={isLight}>{returnShortNameForMetric(metric).name}</Name>
+              <Amount isLight={isLight}>{metric.amount.toLocaleString('es-US')}</Amount>
```

Comment:
We have a helper to avoid using `toLocaleString` directly so we manage the decimal places too: `usLocalizedNumber`

Reply:
you are right, done